controllers/components/dashboard/dashboard_support.go

```diff
@@ -6,24 +6,25 @@ import (
 
```

Comment:
Not for the PR, but in general. Does this file level granularity really make sense?

Reply:
Don't have any strong opinion, the reasoning here was mainly to have all those path/platform-dependant variables in a single place so it is easy to spot misconfiguration and how they relate. 

That said no objections if we find a better place